libs/pilight/protocols/GPIO/ds18b20.c

```diff
@@ -133,6 +138,17 @@ static void *ds18b20Parse(void *param) {
 								}
 								memset(content, '\0', bytes+1);
 
+                                //try to set resolution here - set it temporary to not wear out the EEEPROM
+                                if ((rfd = fopen(ds18b20_w1slave, "w"))) {
+                                    logprintf(LOG_DEBUG, "setting resolution of %s to: %d", ds18b20_w1slave, resolution);
+                                    if (fprintf(ffd, "%d", resolution) < 0 )
+                                        logprintf(LOG_ERR, "cannot set resolution of %s", ds18b20_w1slave);
+
+                                    fclose(ffd);
+                                } else {
+                                    logprintf(LOG_ERR, "opening %s to set resolution failed!", ds18b20_w1slave);
+                                }
+
```

Comment:
spurious blank line

Reply:
You sure about that one?
Removing this blank line  sets the next if right below - kinda destroys readability

Reply:
Ah, never mind then :)